spec/formatting.md

```diff
@@ -235,6 +242,24 @@ rather than the _expression_ in the _selector_ or _pattern_.
 
 _Pattern selection_ is not supported for _fallback values_.
 
+### Expression Attribute Resolution
+
+The only _expression attribute_ that MUST be supported is `@locale`.
```

Comment:
I think this needs to start with the general case before jumping into the built-in attribute(s).

I would also not write this quite this way, in case we add more later. The same effect can be had with:

```suggestion
The _expression attribute_ `@locale` MUST be supported.
```

Or perhaps:

```suggestion
Implementations MUST support the _expression attribute_ `@locale`.
```